File: 3-linked-list/Main.java
class Main {
  public static void main(String[] args) throws Exception {
    SinglyLinkedList sll = new SinglyLinkedList();
    sll.push(11);
    sll.push(12);
    sll.push(13);
    sll.printList();
    sll.removeAtIndex(0);
    sll.printList();
  }
}

File: 3-linked-list/main.cpp
#include <iostream>

#include "SinglyLinkedList.hpp"

using namespace std;

int main(int argc, char const *argv[]) {
  SinglyLinkedList sll;
  sll.push(11);
  sll.push(22);
  sll.push(33);
  sll.push(44);
  sll.removeAtIndex(4);
  sll.printList();
  cout << sll.size() << endl;
  return 0;
}
